### DISCUSSION TITLE
Getting "Activating Extensions" any time I try to do an action related to extensions

### DISCUSSION THREAD

**Discussion by bavjackson** [community/community#67299] [Codespaces]
### Select Topic Area

Bug

### Body

Any time I try to do an action in a codespace related to an extension I get this 'Activating Extensions' spinning in the bottom, but nothing actually happens. Has made codespaces completely unusable, as functionality around port forwarding is totally non functional.

Was getting this issue with a codespace related to a next.js project, but have been able to replicate in a completely new codespace from a blank template.

![ActivatingExtensions](https://github.com/community/community/assets/33766488/42487d7f-5685-40fb-b1be-00cf8e27e1a0)

**Comment by carljonescf** [comment]
I have the same issue. The more complete story...

I've been playing with Codespaces for a couple of days. Following a tutorial online, from my home laptop, I created and ran a Spring Boot application successfully in the browser IDE. All extensions working fine.

I then tried on "onboard" my work laptop to the same workspace. The browser IDE fails in a number of different ways. Sometimes extensions fail to install. When they do install, they do nothing (as per original post, "Activating Extension" spins in the status bar and then disappears). I can't find any logs to provide any diagnostics in the codespace.

Note: I've tried with and without a VPN and Codespaces just spins.

However, I can connect VS Code to the codespace without issue and I'm pretty close to getting IntelliJ connected as well.

More interestingly, I can open the repo in GitPod's browser IDE without issue from the same work network.

I'm guessing that this may be a very subtle firewall issue (the clue being some of the messages in the browser's console), but I could be wrong.

Happy to try to provide more info (tell me what you want).

Thanks
Carl
